### PR TITLE
Adds Ironhammer Ration

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds new ration packs, because the PMCs need their crayon rations
rationpack1

## Why It's Good For The Game
Gives more flavor to IH and Club

## Changelog
:cl:
add: Gave IH their crayons
/:cl:

### CODE REVIEW

**Review by Firefox13 — DISMISSED** [review]
Remove the uwu. I understand that you're trying to pull off them same as ERP joke but this is clearly overdoing it with not only acronym but with the amount of stuff in it too.
Plus, it is filtered out anyway.

**Comment by Firefox13** [comment]
And what's with injectors in MREs that are for containing food?

**Comment by zombiemimic** [comment]
> And what's with injectors in MREs that are for containing food?

Performance drugs

File: code/game/objects/items/weapons/storage/misc.dm [hunk 1]
```diff
@@ -73,6 +73,36 @@
...
+	desc = "Silvery plastic package, with the letters \"IHR\" pasted onto the front. Seems air tight, and vacuumed sealed. \
+	The packaging holds usage information within the fineprint: \
+	\"Instructions: Remove contents from packaging, open ration can, use them in-hand to activate thermal heater. \
+	Thermal insulation will keep them warm for over four hours. Crayons for taste. \
+	Nutrient paste, morale bar, and injector contain medicinal additives for field performace, DO NOT OVERCONSUME.\""
+	can_hold = list(
+		/obj/item/weapon/reagent_containers/food/snacks,
+		/obj/item/weapon/storage/fancy/mre_cracker,
+		/obj/item/weapon/material/kitchen/utensil/spoon/mre,
+		/obj/item/weapon/storage/fancy/crayons,
```
Comment (L74814) [inline code/game/objects/items/weapons/storage/misc.dm:89]: based

File: code/game/objects/items/weapons/storage/misc.dm [hunk 2]
```diff
@@ -73,6 +73,36 @@
...
 
+/obj/item/weapon/storage/ration_pack/ihr
+	icon = 'icons/obj/food.dmi'
+	icon_state = "ihr_closed"
+	name = "ironhammer ration pack"
+	desc = "Silvery plastic package, with the letters \"IHR\" pasted onto the front. Seems air tight, and vacuumed sealed. \
+	The packaging holds usage information within the fineprint: \
+	\"Instructions: Remove contents from packaging, open ration can, use them in-hand to activate thermal heater. \
+	Thermal insulation will keep them warm for over four hours. Crayons for taste. \
+	Nutrient paste, morale bar, and injector contain medicinal additives for field performace, DO NOT OVERCONSUME.\""
```
Comment (Firefox13) [inline code/game/objects/items/weapons/storage/misc.dm:84]:
```suggestion
	Nutrient paste and morale bar medicinal additives for field performace, DO NOT OVERCONSUME.\""
```

File: code/game/objects/items/weapons/storage/misc.dm [hunk 3]
```diff
@@ -73,6 +73,36 @@
...
+	desc = "Silvery plastic package, with the letters \"IHR\" pasted onto the front. Seems air tight, and vacuumed sealed. \
+	The packaging holds usage information within the fineprint: \
+	\"Instructions: Remove contents from packaging, open ration can, use them in-hand to activate thermal heater. \
+	Thermal insulation will keep them warm for over four hours. Crayons for taste. \
+	Nutrient paste and morale bar medicinal additives for field performace, DO NOT OVERCONSUME.\""
+	can_hold = list(
+		/obj/item/weapon/reagent_containers/food/snacks,
+		/obj/item/weapon/storage/fancy/mre_cracker,
+		/obj/item/weapon/material/kitchen/utensil/spoon/mre,
+		/obj/item/weapon/storage/fancy/crayons,
```
Comment (Firefox13) [inline code/game/objects/items/weapons/storage/misc.dm:89]:
```suggestion
		/obj/item/weapon/storage/fancy/crayons
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**